src/main/resources/com/google/api/codegen/nodejs/method_sample.snip

```diff
@@ -11,6 +11,20 @@
         {@initializeRequestObject(apiMethod)}
 
     @end
+    @switch apiMethod.grpcStreamingType.toString
```

Comment:
I think this should be under optionalArrayMethodSampleCode, based on the logic in toolkit, where streaming type is assigned as part of OptionalArrayMethodView